pages/common/ftp.md

```diff
@@ -0,0 +1,23 @@
+# ftp
+
+> Internet File Transfer Program
+
+- Connect to a FTP server:
+
+`ftp {{url}}`
+
+- Switch to binary transfer mode (graphics, compressed files etc):
+
+`binary`
+
+- Turn off prompts for each process:
```

Comment:
For example, if you use `mput` to upload 100 files, it will ask you to confirm every file without this, one by one. _Transfer_ perhaps? Though it goes for stuff like deleting files too. _Interaction_ better maybe?